Metric bootstrapper (#101)
PyTorchLightning/metrics: docs/source/references/modules.rst

~~~~~~~

.. autoclass:: torchmetrics.R2Score
    :noindex:
    
********
Wrappers
********

> Modular wrapper metrics are not metrics in themself, but instead the take in other metrics and alter 

Reviewer: ```suggestion
Modular wrapper metrics are not metrics in themself, but instead take a metric and alter 
```
